### PR TITLE
Fix/types

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Fix staking and sylo types..

### CODE REVIEW

File: src/interfaces/augment-api-tx.ts [hunk 1]
```diff
@@ -762,7 +762,7 @@ declare module '@polkadot/api/types/submittable' {
        * the `origin` falls below minimum bond and is removed lazliy in `withdraw_unbonded`.
        * # </weight>
        **/
-      bond: AugmentedSubmittable<(controller: AccountId | string | Uint8Array, value: Compact<BalanceOf> | AnyNumber | Uint8Array, payee: RewardDestination | { Staked: any } | { Stash: any } | { Controller: any } | { Account: any } | string | Uint8Array) => SubmittableExtrinsic<ApiType>>;
+      bond: AugmentedSubmittable<(controller: AccountId | string | Uint8Array, value: Compact<BalanceOf> | AnyNumber | Uint8Array, payee: RewardDestination | { Stash: any } | { Controller: any } | { Account: any } | string | Uint8Array) => SubmittableExtrinsic<ApiType>>;
```
Comment (holygits) [inline src/interfaces/augment-api-tx.ts:765]: how do we ensure this won't be overwritten when regenerating?

Comment (KarishmaBothara) [reply src/interfaces/augment-api-tx.ts:765]: I haven't changed this manually.. This is auto generated by yarn build after adding RewardDestination in definition.ts for staking..

Comment (holygits) [reply src/interfaces/augment-api-tx.ts:765]: perfect!

File: src/interfaces/sylo/definitions.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default {
     DeviceId: 'u32',
     PreKeyBundle: 'Bytes',
     DeviceIdResponse: 'DeviceId',
-    WithdrawnPreKeyBundle: "('AccountId', 'u32', 'Bytes')",
+    WithdrawnPreKeyBundle: "(AccountId, u32, Bytes)",
```
Comment (holygits) [inline src/interfaces/sylo/definitions.ts:36]: small nit: `'(AccountId, u32, Bytes)'`

**Review by holygits — APPROVED** [review]
lgtm

**Review by alexsednz — APPROVED** [review]

**Pull request merged**